app/views/middleware_server/_add_datasource.html.haml

```diff
@@ -0,0 +1,33 @@
+- @angular_form = true
+
+%ng-form.form-horizontal#form_ds_add{"name"          => "dsAddForm",
+                                     "ng-controller" => "mwAddDataSourceController"}
```

Comment:
@mtho11, testing this, the angular controller is defined as `middlewareAddDataSourceController` but then it's being used here as `mwAddDataSourceController` which causes it to not work at all.